src/content/test/pointer-motion/dragging-movements.tsx

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { create, React } from '../../common';
+
+export const infoAndExamples = create(({ Markup }) => (
+    <>
+        <p>
+            The action of dragging cannot be the only means available to perform an action, with exceptions on where dragging is essential
+            to the functionality, or the dragging mechanism is not built by the web author (e.g., native browser functionality unmodified by
+            the author).
+        </p>
+        <h2>Why it matters</h2>
+        <p>Users who struggle with performing dragging movements need to still operate an interface with a pointer interface.</p>
+        <p>
+            Some people cannot perform dragging movements in a precise manner. Others use a specialized or adapted input device, such as a
+            trackball, head pointer, eye-gaze system, or speech-controlled mouse emulator, which may make dragging cumbersome and
+            error-prone. When an interface implements functionality that uses dragging movements, users perform discrete actions such as
+            tapping or clicking to establish a starting point or press and hold that contact. Not all users can accurately press and hold
+            that contact while also repositioning the pointer. An alternative method must be provided so that users with mobility
+            impairments who use a pointer (mouse, pen, or touch contact) can use the functionality.
+        </p>
+
+        <h2>How to fix</h2>
+        <p>
+            For any function that can be operated through dragging, make sure that the user can (1) operate the function through dragging,
+            or (2) there is a <a href="https://www.w3.org/TR/WCAG22/#dfn-single-pointer">single pointer</a> activation (for example single
```

Comment:
@codeofdusk this anchor link needs to be turned into a `Markup.HyperLink` component so tests pass.
```suggestion
            or (2) there is a <Markup.HyperLink href="https://www.w3.org/TR/WCAG22/#dfn-single-pointer">single pointer</Markup.HyperLink> activation (for example single
```